src/rvm.ts

```diff
@@ -0,0 +1,147 @@
+const options = [
+  {
+    name: ["--version", "-v"],
+    description: "Show version of rvm",
+  },
+  {
+    name: "--default",
+    description:
+      "When used with ruby selection, sets a default ruby for new shells",
+  },
+  {
+    name: "--debug",
+    description: "Enable debug mode",
+  },
+  {
+    name: "--force",
+    description: "Force install, removes old install & source before install",
+  },
+  {
+    name: "--all",
+    description: "Used with 'rvm list' to display 'most' available versions",
+  },
+  {
+    name: "--summary",
+    description: "Used with 'do' to print out a summary of the commands run",
+  },
+  {
+    name: "C",
+    description:
+      "Custom configure options, comma separated, double quote args that need quoting, default",
+  },
+];
+
+const interpreters = [
+  {
+    name: "rbx",
+    description: "Rubinius",
+    options: [...options],
+  },
+  {
+    name: "jruby",
+    description: "JRuby",
+    options: [...options],
+  },
+  {
+    name: "ruby",
+    description: "MRI/YARV Ruby (The Standard), defaults to 1.8.6",
+    options: [...options],
+  },
+  {
+    name: "ree",
+    description: "Ruby Enterprise Edition",
+    options: [...options],
+  },
+  {
+    name: "system",
+    description: "Use the system ruby (eg. pre-RVM state)",
+    options: [...options],
+  },
+  {
+    name: "default",
+    description: "Use RVM set default ruby and system if it hasn't been set",
+    options: [...options],
+  },
+];
+
+const completionSpec: Fig.Spec = {
+  name: "rvm",
+  description: "",
+  options: [
+    {
+      name: ["--help", "-h"],
+      description: "Show help for rvm",
+    },
+  ],
+  subcommands: [
+    {
+      name: "use",
+      description: "Setup current shell to use a specific ruby version",
+      subcommands: [...interpreters],
```

Comment:
Don't need to spread the array here (apply below too)
```suggestion
      subcommands: interpreters,
```